kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -412,9 +422,14 @@ private Node createFieldEntry(PatternField patternField, int fieldNum) {
         commentIconRegion.getStyleClass().add("grey-comment-icon");
         outerHBox.getChildren().addAll(innerHBox, commentIconRegion);
         fieldVBoxContainer.getChildren().addAll(fieldLabel, fieldText, outerHBox);
+        fieldVBoxContainer.setOnMouseClicked(mouseEvent -> showContextMenuPopUp(patternField, fieldVBoxContainer, mouseEvent.getScreenX(), mouseEvent.getScreenY()));
         return fieldVBoxContainer;
     }
 
+    private void showContextMenuPopUp(PatternField patternField, Node node, double x, double y) {
+        contextMenu.show(node, x, y);
+    }
```

Comment:
Use MenuHelper's 
```java
public static void fireContextMenuEvent(ActionEvent actionEvent, Side side, double xOffset, double yOffset)
```

Reply:
This method takes in actionEvent. I am using mouse event for this. I also need to capture the node on which it is clicked. Unlike other fxml code I am create only one ContextMenu.